### PR TITLE
Add support for open folder in VS for local dev

### PR DESCRIPTION
**Pull request by BrennanConroy**
`devenv.exe %folder%` will open the folder and VS will use cmake to build the project.

Requires vcpkg pre-reqs to be done beforehand, but it's a step in the right direction and needed for me to do local dev.

### CODE REVIEW

File: CMakeSettings.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+﻿{
+  "configurations": [
+    {
+      "name": "x64-Debug",
+      "generator": "Ninja",
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${projectDir}\\out\\build\\${name}",
```
Comment (jkotalik) [inline CMakeSettings.json]: Is the folder `out` a standard location for build output?

Comment (BrennanConroy) [reply CMakeSettings.json]: It's "standard" in the sense that it's the default location VS sets up for you

File: signalrclient.sln [hunk 2]
```diff
@@ -1,112 +0,0 @@
-﻿
-Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (jkotalik) [inline signalrclient.sln]: Add a startvs.cmd script to this repo which calls devenv.exe

File: src/signalrclient/CMakeLists.txt [hunk 3]
```diff
@@ -39,4 +39,9 @@ else()
 endif()
 
 include(GNUInstallDirs)
-install(TARGETS signalrclient LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR})
+
+if(WIN32)
```
Comment (jkotalik) [inline src/signalrclient/CMakeLists.txt]: Add a comment on why you need to do this?

**Pull request merged**